cluster/images/etcd/Dockerfile

```diff
@@ -16,4 +16,5 @@ FROM BASEIMAGE
 
 EXPOSE 2379 2380 4001 7001
 COPY etcd* etcdctl* /usr/local/bin/
+RUN chmod +x /usr/local/bin/*
```

Comment:
Can we list specific binaries to chmod? instead of "*"

Reply:
Done, well, mirrored the COPY commands in particular. This results in:
``` shell
$ ls -lah /usr/local/bin
total 242M
drwxrwsr-x. 1 root staff   49 Jul  3 15:45 .
drwxrwsr-x. 1 root staff   17 Feb 28 00:00 ..
-rwxr-xr-x. 1 root root  1.1K Mar 25 17:58 clean-install
-rwx--x--x. 1 root root   19M Jul  3 12:25 etcd
-rwx--x--x. 1 root root   14M Jul  3 12:25 etcd-2.2.1
-rwx--x--x. 1 root root   17M Jul  3 12:25 etcd-2.3.7
-rwx--x--x. 1 root root   20M Jul  3 12:25 etcd-3.0.17
-rwx--x--x. 1 root root   16M Jul  3 12:25 etcd-3.1.12
-rwx--x--x. 1 root root   18M Jul  3 12:25 etcd-3.2.24
-rwx--x--x. 1 root root   19M Jul  3 12:25 etcd-3.3.10
-rwx--x--x. 1 root root   16M Jul  3 12:25 etcdctl
-rwx--x--x. 1 root root   13M Jul  3 12:25 etcdctl-2.2.1
-rwx--x--x. 1 root root   15M Jul  3 12:25 etcdctl-2.3.7
-rwx--x--x. 1 root root   18M Jul  3 12:25 etcdctl-3.0.17
-rwx--x--x. 1 root root   14M Jul  3 12:25 etcdctl-3.1.12
-rwx--x--x. 1 root root   15M Jul  3 12:25 etcdctl-3.2.24
-rwx--x--x. 1 root root   16M Jul  3 12:25 etcdctl-3.3.10
lrwxrwxrwx. 1 root staff    4 Mar 25 17:59 invoke-rc.d -> noop
-rwxr-xr-x. 1 root root   21M Jul  3 15:44 migrate
-rwx--x--x. 1 root root  3.9K Jul  3 15:44 migrate-if-needed.sh
-r-xr-xr-x. 1 root staff    0 Mar 25 17:59 noop
lrwxrwxrwx. 1 root staff    4 Mar 25 17:59 runlevel -> noop
lrwxrwxrwx. 1 root staff    4 Mar 25 17:59 update-rc.d -> noop
```